### ISSUE TITLE
Heatmap does not show any colors if reverse=True

### ISSUE THREAD

**Issue by JoergVanAken** [bloomberg/bqplot#949]
**Describe the bug**
A heatmap does not show any values if `reverse` is set to `True`

**To Reproduce**
The following code:

```
fig = plt.figure()
h = plt.heatmap(np.array([[1, 2], [3, 4]]), y=[0, 1])
plt.show()
```
shows up a heatmap. But after setting

`h.scales['x'].reverse=True`

the axis is reversed but all colors disappear,  (changing `y` has the same effect)

**Expected behavior**
The colur should not disappear.

**Environment (please complete the following information):**
 - Operating System and Version: Windows 10
 - Browser: Chrome

**Comment by martinRenou** [comment]
Thanks a lot for filling this issue, I'm looking into this

**Comment by SergioLoaiza** [comment]
**Describe the bug**
The bug seems to be more general. I experienced the same bug creating a simple bar graph; the bars disappeared if `reverse` is set to `True` at a `LinearScale`.

**To Reproduce**
```python
from bqplot import OrdinalScale, Bars, LinearScale

quintes = np.array(
    ["0 - 4", "5 - 9", "10 - 14", "15 - 19", "20 - 24", "25 - 29", "30 - 34", "35 - 39",
     "40 - 44", "45 - 49", "50 - 54", "55 - 59", "60 - 64", "65 - 69", "70 - 74",
     "75 - 79", "80 - 84", "85 - 89", "90 - 94", "95 - 99", "> 100"])

men = np.array(
    [1915887, 2100931, 2494484, 2464805, 2361297, 2529633, 2669927, 2754129, 2753282,
     2552531, 2211649, 1697221, 1311024, 902675, 722246, 482686, 273915, 108639, 35867,
     10965, 1238])

x_ord = OrdinalScale()
y_linear = LinearScale()

bars = Bars(x=quintes, y=men,
           scales={'x':x_ord, 'y': y_linear},
           orientation="horizontal",
           colors=["deepskyblue"]
           )

# bars.scales['y'].reverse=True

ax_x = Axis(scale=x_ord, orientation = 'vertical', visible=False)

ax_y = Axis(scale=y_linear, tick_format="0.0f")

fig = Figure(marks=[bars], axes=[ax_x, ax_y])

fig
```

The bars graph is displayed. However, after uncommenting this line in the code:
```python
bars.scales['y'].reverse=True
```
the bars disappear even though the axes is effectively reversed.

**Expected behavior**
The bars should be displayed in the same way of the axes, that is, from right to left.

**Environment (please complete the following information):**
Ubuntu 21.04
Python 3.9.5
JupyterLab 3.2.4
bqplot 0.12.32

**Comment by martinRenou** [comment]
Thanks for commenting, would you like to open a new issue for that specifically?

**Comment by SergioLoaiza** [comment]
Thank you for your advice. I published it again in this new issue [#1483 ](https://github.com/bqplot/bqplot/issues/1483)
